config/cluster.yaml

```diff
@@ -22,4 +22,4 @@ docker_repo: jiata/thunderbolt:0.1.0-spark2.2.0-python3.5.4
 
 
 # wait: <true/false> 
-wait: true
+wait: false
```

Comment:
I think JS and I had another conversation on this and kept flip-flopping.
@jiata  - Where should we land on this for our initial release? I think we finally agreed on 'wait: true'

Reply:
yeah wait = True

Reply:
Okay, then i'll just close this branch.